Classes/Projection/Doctrine/DoctrineProjectionPersistenceManager.php

```diff
@@ -129,7 +138,7 @@ public function persistAll()
 
         try {
             $this->entityManager->flush();
-        } catch (\Exception $exception) {
+        } catch (ORMException $exception) {
```

Comment:
This is not exactly related, but it avoids exceptions from being swallowed..
I.e. I had some duplicate primary keys, but the CLI only died with `EntityManager closed`